tests/pootle_fs/utils.py

```diff
@@ -167,10 +167,10 @@ def test_project_fs_instance_bad(english):
     project.config["pootle_fs.fs_type"] = "foo"
     with pytest.raises(NotConfiguredError):
         FSPlugin(project)
-    project.config["pootle_fs.fs_type"] = None
-    project.config["pootle_fs.fs_url"] = "bar"
+    project.config["pootle_fs.fs_type"] = "localfs"
     with pytest.raises(NotConfiguredError):
         FSPlugin(project)
+    project.config["pootle_fs.fs_url"] = "/bar"
```

Comment:
Nit: Shouldn't the `fs_url` be set after setting `fs_type`?

Reply:
this test is a bit messy - im gonna clean it up after looking at validation of the fs_type